variables.tf

```diff
@@ -57,11 +57,11 @@ variable "cos_location" {
 }
 
 variable "cos_plan" {
-  description = "The plan to use when Object Storage instances are created. Possible values: `standard`, `lite`, `cos-one-rate-plan`. Applies only if `create_cos_instance` is true."
+  description = "The plan to use when Object Storage instances are created. Possible values: `standard`, `cos-one-rate-plan`. Applies only if `create_cos_instance` is true. The standard plan includes a Free Tier of 5GB free storage for 12 months. To access the Free Tier select 'smart' as your bucket storage class."
```

Comment:
I think just point to this doc instead of duplicating it in the description. The details may drift over time, so better to just point to the doc thats maintain by COS team https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-provision